### PR TITLE
feat: print env config to log at startup

### PR DESCRIPTION
**Pull request by dkhokhlov**
Ref: LOG-18170

### CODE REVIEW

File: common/config/src/env_vars.rs [hunk 1]
```diff
@@ -1,3 +1,8 @@
+use lazy_static::lazy_static;
+
+//
+// DO NOT FORGET to add new consts to ENV_VARS_LIST at the end file.
```
[the remark this answers is not in the document]
Comment (dkhokhlov) [reply common/config/src/env_vars.rs:4]: what do you mean by insert the value in the ENV_VARS_LIST?

[the remark this answers is not in the document]
Comment (jakedipity) [reply common/config/src/env_vars.rs:4]: Yeah this is bound to fall out of sync so we should look for a more tied up solution

[the remark this answers is not in the document]
Comment (dkhokhlov) [reply common/config/src/env_vars.rs]: changed

File: common/config/src/env_vars.rs [hunk 2]
```diff
@@ -1,40 +1,68 @@
...
-pub const LINE_INCLUSION: &str = "MZ_LINE_INCLUSION_REGEX";
-pub const REDACT: &str = "MZ_REDACT_REGEX";
-pub const INGEST_TIMEOUT: &str = "MZ_INGEST_TIMEOUT";
-pub const INGEST_BUFFER_SIZE: &str = "MZ_INGEST_BUFFER_SIZE";
-pub const RETRY_DIR: &str = "MZ_RETRY_DIR";
-pub const RETRY_DISK_LIMIT: &str = "MZ_RETRY_DISK_LIMIT";
-pub const INTERNAL_FS_DELAY: &str = "MZ_INTERNAL_FS_DELAY";
-pub const CLEAR_CACHE_INTERVAL: &str = "MZ_CLEAR_CACHE_INTERVAL";
-pub const METADATA_RETRY_DELAY: &str = "MZ_METADATA_RETRY_DELAY";
+macro_rules! define_env_var {
```
[the remark this answers is not in the document]
Comment (dkhokhlov) [reply common/config/src/env_vars.rs:1]: we need it

**Comment by dkhokhlov** [comment]
@c-nixon 
this trivial harmless change broke properties test!

**Pull request merged**